packages/react-examples/src/react-badge/PresenceBadge/PresenceBadge.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { PresenceBadge } from '@fluentui/react-badge';
+
+export const PresenceBadgeStatus = () => (
+  <>
+    <h2>Circular (default)</h2>
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        width: 200,
+      }}
+    >
+      <PresenceBadge status="busy" />
+      <PresenceBadge status="away" />
+      <PresenceBadge status="available" />
+      <PresenceBadge status="outOfOffice" />
+      <PresenceBadge status="offline" />
```

Comment:
Eventually it'd be good to have an example for all of the variations outlined in the visual spec. That could be done in a follow-up PR though.

Reply:
Added all status and OOF